task5.py

```diff
@@ -0,0 +1,8 @@
+vip_room = {'place1': 'David', 'place2': 'Ketrin', 'place3': 'Misha', 'place4': 'Mark'}
+room = {'place5': 'Peter', 'place6': 'Maria', 'place7': None, 'place8': None}
```

Comment:
dict is not the best way to store this data. Because dict is mutable. In real life you can not extend number of VIP guests of common guests in the cinema